### PR TITLE
added feature to have custom header, custom week names, custom markin…

### PR DESCRIPTION
**Pull request by trickyc0d3r**
…g of custom week days

### CODE REVIEW

File: example/android/.settings/org.eclipse.buildship.core.prefs [hunk 1]
```diff
@@ -0,0 +1,2 @@
+connection.project.dir=
+eclipse.preferences.version=1
```
Comment (friederbluemle) [inline example/android/.settings/org.eclipse.buildship.core.prefs]: Please remove this file (as well as `example/android/.project`) - They should not be in source control.

Comment (trickyc0d3r) [reply example/android/.settings/org.eclipse.buildship.core.prefs]: oh, my bad. ok. sorry, I don't know why they appear every-time

File: src/calendar/index.js [hunk 2]
```diff
@@ -272,6 +272,7 @@ class Calendar extends Component {
           weekNumbers={this.props.showWeekNumbers}
           onPressArrowLeft={this.props.onPressArrowLeft}
           onPressArrowRight={this.props.onPressArrowRight}
+          {...this.props}
```
Comment (Inbal-Tish) [inline src/calendar/index.js]: This is dangerous as you let the user override all props sent to CalendarHeader.

File: src/calendar/header/index.js [hunk 3]
```diff
@@ -130,7 +130,7 @@ class CalendarHeader extends Component {
 
     return (
       <View style={this.props.style}>
-        <View style={this.style.header}>
+        {this.props.customHeader ? this.props.customHeader() : <View style={this.style.header}>
```
Comment (Inbal-Tish) [inline src/calendar/header/index.js]: I would prefer a check if 'customHeader' is even a function, or safely invoke using lodash

File: src/calendar/header/index.js [hunk 4]
```diff
@@ -87,7 +87,7 @@ class CalendarHeader extends Component {
   render() {
     let leftArrow = <View />;
     let rightArrow = <View />;
-    let weekDaysNames = weekDayNames(this.props.firstDay);
+    let weekDaysNames = this.props.customDays ? this.props.customDays: weekDayNames(this.props.firstDay);
```
Comment (Inbal-Tish) [inline src/calendar/header/index.js]: The prop 'customDays' is not declared in the propTypes

File: example/android/.settings/org.eclipse.buildship.core.prefs (hunk 1 above)
Comment (friederbluemle) [reply example/android/.settings/org.eclipse.buildship.core.prefs]: The reason they (re-)appear is a buggy Java plugin for VS Code. Unless you heavily rely on it, the recommendation is to uninstall it.
Also see: https://github.com/redhat-developer/vscode-java/issues/618

**Pull request closed without merging**